RELEASES.md

```diff
@@ -6,53 +6,93 @@ Below are the versioning schemes used for publishing debs:
-```
- 11. Create a merge proposal such as [24.2 PR](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/385073), [24.3 PR](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/389745)
- 11. For SRUs: Create an "upload a new version bug" in https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bugs
- 12. Describe the need, provide testing PPA and describe test instructions
- 13. Ping appropriate maintainer about upload and verification
- 14. Validate tests, accept upload and ship it
-
-Previous release bugs:
-| ------- |
-| [20.3 Focal](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1869980) |
-
+ 9. Create a merge proposal for 27.1 which targets `ubuntu/devel`
+     * For an example, see the [27.0.2 merge proposal](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/402459)
+ 10. Add 2 review slots for `canonical-server` and `canonical-server-core-reviewers`
+
+## SRU Release Process
+Below is the procedure used to SRU ubuntu-advantage-client to a stable Ubuntu series. It makes the following assumptions:
+* The procedure for the Devel Release Process has just completed
+* A local git branch representing the MP most recently released to the Ubuntu devel release is `release-27.1-impish`
+
+ 1. Create an "upload a new version bug" in https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bugs
+   * Describe the need, provide testing PPA and describe test instructions
+   * For examples, see [27.0](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1926361) or [20.3](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1869980)
+ 2. Create a PR for each target series based off your local `release-${UA_VERSION}-impish` branch:
+ ```bash
+ UA_VERSION=27.1
+ SRU_BUG=TODO_FROM_STEP_11
+ rm -rf ../out
+ for release in xenial bionic focal groovy hirsute do
+  git checkout release-${UA_VERSION}-impish -B upload-${UA_VERSION}-$release
+  case "${release}" in
+      xenial) version=${UA_VERSION}~16.04.1;;
+      bionic) version=${UA_VERSION}~18.04.1;;
+      focal) version=${UA_VERSION}~20.04.1;;
+      groovy) version=${UA_VERSION}~20.10.1;;
+      hirsute) version=${UA_VERSION}~21.04.1;;
+  esac
+  dch -v ${version} -D ${release} Backport new upstream release: (LP: #${SRU_BUG}) to $release
+  git commit -m 'changelog backport to ${release}' debian/changelog
+  git push <your_launchpad_username> upload-${UA_VERSION}-$release
+  build-package # to create release-specific dsc files used to upload in step 16
+ ```
+ 3. Create merge proposals for each SRU target release @ `https://code.launchpad.net/~<YOUR_LP_USER></YOUR_LAUNCHPAD_USER>/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/`. Make sure each PR targets your `release-${UA_VERSION}-impish` branch.
```

Comment:
The PR should target `impish` branch ? Shouldn't we target the devel branches for each release ?

Reply:
@lucasmoura this was decided as part of both the upload  and the SRU review is that it makes it easier to review given that we are syncing the exact same content (minus a changelog diff) into all supported Ubuntu releases. The upload reviewer will be able to handle the upload separately and not actually merge the releases into the impish branch as part of the review.

Reply:
Oh okay, sorry, I forgot about that part. My mistake